plugins/vpp/ifplugin/vppcalls/dump_bfd_vppcalls.go

```diff
@@ -0,0 +1,193 @@
+// Copyright (c) 2018 Cisco and/or its affiliates.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package vppcalls
+
+import (
+	"net"
+	"time"
+
+	bfdapi "github.com/ligato/vpp-agent/plugins/vpp/binapi/bfd"
+	"github.com/ligato/vpp-agent/plugins/vpp/model/bfd"
+)
+
+// BfdDetails is the wrapper structure for the BFD northbound API structure.
+type BfdDetails struct {
+	Bfd  *bfd.SingleHopBFD
```

Comment:
This will need struct tags for json to make it consistent. This applies to some of the others as well.